frontend/public/locales/en/demographics-oral-health-questions.json

```diff
@@ -6,6 +6,9 @@
 
     "question1": "1. Where were you born?",
 
+    "question3": "3. Which of the following best describes your racial or ethnic group? Check all that apply:",
```

Comment:
These questions should probably be rendered as an `<ol>`, instead of putting the numbering inside of the i18n files.

Since you were just following convention by adding a new entry, I think this change can be done in a future PR.

Reply:
Sounds reasonable. Someone can take care of that after the full list of questions are done.

Reply:
Resolving as [AB#3034](https://dev.azure.com/DTS-STN/Canada%20Dental%20Care%20Plan/_workitems/edit/3034) has been created to address this.